elasticsearch/src/main/scala/akka/stream/alpakka/elasticsearch/ElasticsearchFlowStage.scala

```diff
@@ -23,33 +23,58 @@ import akka.stream.alpakka.elasticsearch.scaladsl.ElasticsearchSinkSettings
 import org.apache.http.message.BasicHeader
 import org.apache.http.util.EntityUtils
 
-final case class IncomingMessage[T](id: Option[String], source: T)
+object IncomingMessage {
+  // Apply method to use when not using cargo
+  def apply[T](id: Option[String], source: T): IncomingMessage[T, Any] =
```

Comment:
It should be used `NotUsed` instead of `Any`.

Reply:
done